lib/tapioca/loaders/loader.rb

```diff
@@ -103,30 +145,27 @@ def safe_require(path)
       sig { void }
       def silence_deprecations
         # Stop any ActiveSupport Deprecations from being reported
-        Object.const_get("ActiveSupport::Deprecation").silenced = true
+        if defined?(ActiveSupport::Deprecation)
+          ActiveSupport::Deprecation.silenced = true
+        end
       rescue NameError
         nil
```

Comment:
I don't think we can still get a `NameError` in this method, due to the `defined?` check. So we can remove this.